enhancements/update/distribute-secondary-metadata-as-container-image.md

```diff
@@ -0,0 +1,185 @@
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+### Content trust
+Trust in the container image registry and in the container image content is possibly an issue if we follow an approach where Cincinnati uses a mutable tag, e.g. `latest` tag to receive the most recent content from the container registry.
+Assuming we're using Quay as an image registry, [which does not support Docker Content Trust](https://support.coreos.com/hc/en-us/articles/115000289114-Docker-Content-Trust-support).
+If we decide to ensure image integrity, which means do we have that do not further complicate the workflow?
+
+### Product name for the CPaaS integration
+Which product name should this image be released under?
+Related to this is that it's currently unclear if the Cincinnati image itself will be published under its own product name or now.
+Further note: the repository for the graph-data images will not be semantically versioned.
+
+### Repository location and name
+Under which well-known container image repository will we publish the images?
+
+### Continuous build and release
+Can CPaaS handle continuous build and release from master?
+
+## Summary
+The [cincinnati-graph-data GitHub repository][cincinnati-graph-data] will be continuously packaged and released in form of a container image by the means of integration with CPaaS.
+This image will be available for public consumption, and will be used by the publicly available Cincinnati instances operated by Red Hat.
+The image can be transferred and served to Cincinnati instances operated by any party, including but not limited to network-restricted environments.
+The source of the container image, namely the cincinnati-graph-data, will remain publicly available and its workflows unaffected by this enhancement.
+
+## Motivation
+
+### Goals
+1. The container image will become the primary way of consuming and distributing the content of the cincinnati-graph-data repository.
+2. As long as the primary release metadata will be packaged, distributed and consumed in form of container images as well, the transport mechanism required for data consumption by Cincinnati, as well as mirroring and serving its consumed content, will be solely [Docker Registry HTTP API V2][docker-registry-http-api-v2] protocol.
+
+
+### Non-Goals
+1. Changes to the GitHub repository workflow or data format.
+2. Implementation details about the streamlining of the mirroring process for primary and secondary release metadata.
+
+## Proposal
+
+### User Stories
+
+#### Content mirroring for Cincinnati deployments
+A user might want to mirror all content which is required by Cincinnati.
+
+1. The user sets up a container image registry in the target environment.
+2. Optionally, if the target environment does not have internet access, the user downloads the release container images and the desired graph-data container image to a transfer medium.
```

Comment:
Optionally, if the target environment does not have internet access, the user downloads the release container images and the desired graph-data container image to a transfer medium.
```suggestion
2. Optionally, if the target environment does not have internet access, the user downloads the release container images, the desired graph-data container image and the image's verification signature to a transfer medium .
```